config.json

```diff
@@ -653,6 +653,14 @@
         "prerequisites": [],
         "difficulty": 3
       },
+      {
+        "slug": "pop-count",
+        "name": "Pop Count",
+        "uuid": "bdf29951-ece0-4212-ae1f-c4a09359aa45",
+        "practices": ["numbers", "while-loops"],
```

Comment:
I think just putting "bit manipulation" as practice is better since we already have quite a few exercises practicing numbers and while loops.

Reply:
I wasn't sure about this one as the "bit manipulation" concept hasn't been written yet. Originally, I was thinking of doing the concept and updating this `practice` array together in a later PR. I'm happy to still change this now or add the concept to this PR if either is still preferred.

Reply:
I think doing it now is better and then we can add a concept later.